community-operators/microcks/microcks-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,197 @@
+      mediatype: image/png 
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: microcks-ansible-operator
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - microcks.github.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+      deployments:
+      - name: microcks-ansible-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: microcks-ansible-operator
+          template:
+            metadata:
+              labels:
+                name: microcks-ansible-operator
+            spec:
+              serviceAccountName: microcks-ansible-operator
+              containers:
+                - name: microcks-ansible-operator
+                  image: quay.io/microcks/microcks-ansible-operator
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
```

Comment:
If this operator supports AllNamespaces InstallMode for watching CRs clusterwide and the downward API as defined by OLM, then this should be set to `metadata.annotations['olm.targetNamespaces']`

Reply:
Right. Fix installNodes in commit e49983af1e6a18db2c377218a87eab53b9235bd7 .

Reply:
This should still be set to `metadata.annotations['olm.targetNamespaces']`

Reply:
Now done in 238ce6e